Rocket.Chat/Controllers/Chat/MembersListViewController.swift

```diff
@@ -192,3 +192,28 @@ extension MembersListViewController: UITableViewDelegate {
         }
     }
 }
+
+extension MembersListViewController {
+    func checkUserBlockState(_ user: User) {
+        if AuthManager.isAuthenticated()?.canUnblockUser(user) == .allowed {
+            let alert = UIAlertController(title: nil, message: nil, preferredStyle: .actionSheet)
+            alert.addAction(UIAlertAction(title: localized("chat.member.unblock.title"), style: .default, handler: { [weak self] _ in
+                self?.delegate?.membersList(self!, didSelectUser: user)
```

Comment:
This title doesn't look like makes sense on the action handler. 🤔

Reply:
Also, please don't use `self!`. It's a weak variable.

Reply:
Ok, I will change it and push after my classes! Thank you!